docker/build/Dockerfile.export

```diff
@@ -3,6 +3,7 @@ FROM golang:1.11.4-alpine
 RUN apk add --no-cache gcc musl-dev
 
 ADD ./vendor/github.com/orbs-network/orbs-contract-sdk/go/ /go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/orbs-contract-sdk/go/
+ADD ./vendor/github.com/orbs-network/contract-external-libraries-go/ /go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/contract-external-libraries-go/
```

Comment:
should this be here AND in the experimental file?

Reply:
Fixed.